umi_tools/dedup.py

```diff
@@ -553,7 +554,7 @@ def get_bundles(insam, ignore_umi=False, subset=None, quality_threshold=0,
             if whole_contig:
                 do_output = not read.tid == last_chr
             else:
-                do_output = start > (last_pos+1000) or not read.tid == last_chr
+                do_output = start > (last_pos+1000) and not read.tid == last_chr
```

Comment:
Apologies for this. I had a similar issue with the changes you made to counting outputted reads - 968ee3be238263dcf0285a58b419ccf57f977614. I'm not sure how this happens given the branch was made from master after these commits?